src/concurrency/ocf_cache_line_concurrency.c

```diff
@@ -815,6 +828,9 @@ static int _ocf_req_lock_rd(struct ocf_request *req, ocf_req_async_lock_cb cb)
 
 err:
 	for (; i >= 0; i--) {
+		if (req->map[i].status <= LOOKUP_MISS) {
```

Comment:
Don't you think `req->map[i].status == LOOKUP_MISS || req->map[i].status == LOOKUP_HIT` would be more readable?

Reply:
this is an error, I meant to use 
if (!_ocf_req_needs_cl_lock(req, i))